scout/server/extensions/loqus_extension.py

```diff
@@ -148,6 +151,35 @@ def get_exec_loqus_version(self, loqusdb_id=None):
         LOG.debug("version: {}".format(version))
         return version
 
+    def set_loqus_query(self, variant_obj, category):
```

Comment:
This code was in the variant controller before, but I think this is a good place to have this function instead!